cypress/support/commands.js

```diff
@@ -23,3 +23,19 @@
 //
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+
+Cypress.Commands.add('login', (username, password) => {
+  cy.request({
+    method: 'POST',
+    url: 'https://demoqa.com/Account/v1/Login', // baseUrl is prepend to URL
+    body: {
+      userName: "serhio1234", 
+      password: "1234Serhio!"
+    },
+  }).then((response) => {
+    cy.setCookie('token', response.body.token);
+    cy.setCookie('userName', response.body.username);
+    cy.setCookie('userID', response.body.userId);
+    cy.setCookie('expires', response.body.expires);
+  })
+})
\ No newline at end of file
```

Comment:
Please add a newline at the end of the file to avoid potential issues with file formatting. ↩️